### PR TITLE
Bookstore

### PR DESCRIPTION
**Pull request by Emmy-github-webdev**
# Bookstore

In this project, I

- Initialise React app
- Follow Gitflow 
- Ensure there are no lint errors
- Add Redux
- Follow the feature folder approach
- Implement reusable components
- Add Router and set Route and Links for Books, Categories, and Home page

### CODE REVIEW

File: src/components/HomePage.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function HomePage() {
+  return (
+    <div>
+      Welcome to homepage
+    </div>
+  );
```
Comment (dasileker) [inline src/components/HomePage.js]: - [ ] you should have an array of objects then you call them props in the books.js.

ppp

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @Emmy-github-webdev,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/HomePage.js (hunk 1 above)
Comment (Emmy-github-webdev) [reply src/components/HomePage.js]: Thanks, @dasileker, recommendation implemented.

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @Emmy-github-webdev , 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Emmy-github-webdev** [comment]
Thank you @Mihndim2020

**Pull request merged**